canvas_modules/harness/cypress/support/properties/properties-cmds.js

```diff
@@ -53,3 +34,43 @@ Cypress.Commands.add("openSubPanel", (title) => {
 	cy.get(".properties-icon-button-label").contains(title)
 		.click();
 });
+
+Cypress.Commands.add("clickAction", (actionName) => {
+	cy.get(".properties-action-button button").contains(actionName)
+		.click();
+});
+
+Cypress.Commands.add("clickSubPanelButtonInRow", (controlId, row) => {
+	cy.get("div[data-id='properties-" + controlId + "']").find(".properties-subpanel-button")
+		.then((idx) => {
+			idx[row].click();
+		});
+});
+
+Cypress.Commands.add("setRenameSubPanelLabel", (controlId, labelText) => {
```

Comment:
I think this is just a text input so we can just make a generic 
setTextFieldValue... method.